packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1586,86 +1612,72 @@ class _ActionSheetActionSection extends StatelessWidget {
 }
 
 // The part of an action sheet without the cancel button.
-class _ActionSheetMainSheet extends StatefulWidget {
+//
+// The reason that this class is not merged into `_CupertinoActionSheetState` is
+// because a few old unit tests query this class to verify layout behavior.
```

Comment:
I think it is generally preferable to separate out state and presentation widget like this. but I don't think doing this due to old unit tests is good reasoning. I will probably just remove this comment.